packages/thyra/adapters/tpetra/src/Thyra_TpetraVectorSpace_decl.hpp

```diff
@@ -46,6 +46,7 @@
 
 #include "Thyra_SpmdVectorSpaceDefaultBase.hpp"
 #include "Tpetra_Map.hpp"
+#include "Tpetra_MultiVector.hpp"
```

Comment:
It looks like you only need forward declarations for Map and MultiVector in this file.  You can get those from `Tpetra_Map_fwd.hpp` and `Tpetra_MultiVector_fwd.hpp`.

Reply:
Ok, will do.